internal/controller/nimcache_controller.go

```diff
@@ -1088,7 +1088,7 @@ func (r *NIMCacheReconciler) constructJob(ctx context.Context, nimCache *appsv1a
 					},
 					{
 						Name:  "NGC_HOME",
-						Value: "/model-store/ngc",
+						Value: "/model-store/ngc/hub",
```

Comment:
i wonder if HF_HOME also needs to be changed for downloading vllm profile. Please double check. Also, verify that the cache is getting re-used with service deployments with this change.